src/container/mobileNavigation.jsx

```diff
@@ -23,6 +23,7 @@ export const MobileNavigation = () => {
                     <NavigationElement icon={faBars} title="Meny" onClick={() => setShowSidebar(!showSidebar)} />
                     <NavigationSpacer />
                     <NavigationElement icon={faUser} title="Min bruker" onClick={() => history.push(`/user/${auth.authUser.uuid}`)} />
+                    <NavigationElement icon={faSignOutAlt} title="Logg ut" onClick={() => auth.logout()} />
```

Comment:
Is this visible on desktop?

Reply:
This is not visible on desktop, only visible for mobile users.
Everything inside `MobileNavigation` is only shown to mobile users.

Desktop users have this button instead
![image](https://github.com/phoenixlan/phoenixcrew-v1/assets/30247084/959b678a-3ef6-4e36-90d1-44de0b7cbc85)